test/sql/test_selectable.py

```diff
@@ -1137,6 +1137,26 @@ def test_whereclause_adapted(self):
         assert s3._whereclause.left.table is not s1
         assert s3._whereclause.left.table in s3._froms
 
+    def test_values(self):
+        v = Values(
```

Comment:
**mike bayer** (zzzeek) wrote:

general compile tests for the moment should be in test/sql/test_compiler.py.  here, we can do assert_compile() both with and without literal_binds=True to see the difference in rendered behavior. Also, the assert_compile() calls here should include checkparams={...} so that we assert the correct parameters are set up.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/1608

Reply:
**Gord Thompson** (gordthompson) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/1608